### PR TITLE
bundle: add new OLM dependencie ceph csi operator

### PR DESCRIPTION
**Pull request by iamniting**
Signed-off-by: Nitin Goyal <nigoyal@redhat.com>

### CODE REVIEW

File: hack/make-bundle-vars.mk [hunk 1]
```diff
@@ -99,6 +99,13 @@ CSIADDONS_BUNDLE_IMG_TAG ?= v0.8.0
 CSIADDONS_BUNDLE_IMG_LOCATION ?= quay.io/csiaddons
 CSIADDONS_BUNDLE_IMG ?= $(CSIADDONS_BUNDLE_IMG_LOCATION)/$(CSIADDONS_BUNDLE_IMG_NAME):$(CSIADDONS_BUNDLE_IMG_TAG)
 
+CEPHCSI_BUNDLE_NAME ?= cephcsi-operator
+CEPHCSI_BUNDLE_IMG_NAME ?= $(CEPHCSI_BUNDLE_NAME)-bundle
+CEPHCSI_BUNDLE_VERSION ?= v4.17.0
+CEPHCSI_BUNDLE_IMG_TAG ?= main-5074691
+CEPHCSI_BUNDLE_IMG_LOCATION ?= quay.io/ocs-dev
+CEPHCSI_BUNDLE_IMG ?= $(CEPHCSI_BUNDLE_IMG_LOCATION)/$(CEPHCSI_BUNDLE_IMG_NAME):$(CEPHCSI_BUNDLE_IMG_TAG)
```
[the remark this answers is not in the document]
Comment (iamniting) [reply hack/make-bundle-vars.mk:107]: Madhu it is done like this for all other operators. I would like to keep the same for this too.

File: hack/make-bundle-vars.mk [hunk 2]
```diff
@@ -162,6 +169,13 @@ CSIADDONS_SUBSCRIPTION_STARTINGCSV ?= $(CSIADDONS_BUNDLE_NAME).$(CSIADDONS_BUNDL
 CSIADDONS_SUBSCRIPTION_CATALOGSOURCE ?= $(OPERATOR_CATALOGSOURCE)
 CSIADDONS_SUBSCRIPTION_CATALOGSOURCE_NAMESPACE ?= $(OPERATOR_CATALOGSOURCE_NAMESPACE)
 
+CEPHCSI_SUBSCRIPTION_NAME ?= $(CEPHCSI_BUNDLE_NAME)
+CEPHCSI_SUBSCRIPTION_PACKAGE ?= $(CEPHCSI_BUNDLE_NAME)
+CEPHCSI_SUBSCRIPTION_CHANNEL ?= $(DEFAULT_CHANNEL)
+CEPHCSI_SUBSCRIPTION_STARTINGCSV ?= $(CEPHCSI_BUNDLE_NAME).$(CEPHCSI_BUNDLE_VERSION)
+CEPHCSI_SUBSCRIPTION_CATALOGSOURCE ?= $(OPERATOR_CATALOGSOURCE)
+CEPHCSI_SUBSCRIPTION_CATALOGSOURCE_NAMESPACE ?= $(OPERATOR_CATALOGSOURCE_NAMESPACE)
```
[the remark this answers is not in the document]
Comment (iamniting) [reply hack/make-bundle-vars.mk:177]: same here

**Review by Nikhil-Ladha — APPROVED** [review]
/lgtm

**Pull request merged**